core/templates/components/question-directives/question-editor/question-editor.component.ts

```diff
@@ -60,6 +60,7 @@ export class QuestionEditorComponent implements OnInit, OnDestroy {
   @Input() question!: Question;
   @Input() questionId!: string;
   @Input() questionStateData!: State;
+  @Input() stateContentShouldStayVisibleOnScroll!: boolean;
```

Comment:
If this input is required, what about [other usages of the question-editor](https://github.com/search?q=repo%3Aoppia%2Foppia+%3Coppia-question-editor&type=code)? In my understanding you should pass this argument there too.